tool/tsh/mfa.go

```diff
@@ -207,7 +208,7 @@ func newMFAAddCommand(parent *kingpin.CmdClause) *mfaAddCommand {
 	c.Flag("name", "Name of the new MFA device").StringVar(&c.devName)
 	c.Flag("type", fmt.Sprintf("Type of the new MFA device (%s)", strings.Join(defaultDeviceTypes, ", "))).
 		EnumVar(&c.devType, defaultDeviceTypes...)
-	if wancli.IsFIDO2Available() {
+	if wancli.IsFIDO2Available() || winwebauthn.IsAvailable() {
```

Comment:
Do we want to add a higher-level check to wancli? Should IsFIDO2Available return true if the Windows impl is present? Strictly speaking if we have winwebauthn we do have FIDO2.

Reply:
that's good question, I was also thinking about some higher-level check but did not come with good name. Do you have any good suggestions? `wancli.IsPasswordlessCapableClient`?

Reply:
Looking at the usages, I think we could do 2 separate methods:

```go
package webauthncli

func isLibfido2Enabled() bool {...} // same as the "old" IsFIDO2Available, now private

func IsFIDO2Available() bool { return isLibfido2Enabled() || winwebauthn.IsAvailable() }
```

Touch ID remains as a separate check, as it's a separate device type.

Reply:
I am not sure if we can make IsFIDO2Available change how it works. It is used in other places which clearly works only with libfido2 library. 
Maybe we can done something like that:
```
IsFIDO2Available  ->> IsLibfido2Available
IsPasswordlessSelectable() bool { return  IsLibfido2Available || winwebauthn.IsAvailable()}
```

Reply:
good idea, i misread first comment but now it's clear. Changes already pushed.